cytotable/convert.py

```diff
@@ -349,7 +319,7 @@ def _source_chunk_to_parquet(
     # create selection statement from lists above
     select_columns = ",".join(
         # if we should sort the output, add the metadata_cols
-        cytotable_metadata_cols + casted_source_cols
+        casted_source_cols
```

Comment:
How are metadata columns added now with this change?

Reply:
Thanks for asking about this, CytoTable-specific metadata columns (columns which add CytoTable data) are no longer added to tables. Metadata columns from the source data (for ex. from CellProfiler) will remain unchanged in terms of how they are processed by CytoTable.